M30 New composer documentation (#239)
exoplatform/exo-documentation: docs/GettingStarted.rst

Posting in the activity stream via the New Composer
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~	

> A new way to share in the activity stream is available via a clickable text "Post" from the home page and "Post in '$Space Name' " from space's home page.

Reviewer: I don't see any changes for this label in the code.

Author: these changes are not done (the tasks have been created today) but i changed the text to prevent this

|image220|

|image238|

Once you click on the text, a drawer allowing you to type and post a short message is displayed. You can write up to 1300 characters and post to your connections or a space's audience.

|image221|

> If you are typing your message in a space and you have the 

Reviewer: the what ? :smile:

Author: :D I forgot to delete this sentence :D

> In case you exceed the maximum number of characters. the characters counter colour changes to red and the post button is disabled.

Reviewer: ```suggestion
In case you exceed the maximum number of characters, the characters counter colour changes to red and the post button is disabled.
```

|image241|

> If you are typing your message in a space for which you are manager or if you have the rôle redactor or publisher, when you exceed the maximum number of characters, a clickable text "Switch to an article" appears under the ck editor area.

Reviewer: ```suggestion
If you are typing your message in a space for which you are manager or if you have the role redactor or publisher, when you exceed the maximum number of characters, a clickable text "Switch to an article" appears under the editor area.
```

|image240|

> You can delete a folder from the select folder drawer by right clicking on the folder icon then choosing the delete option. You just have to confirm the action from the confirmation popup to delete definetely the selected folder.

Reviewer: ```suggestion
You can delete a folder from the select folder drawer by right clicking on the folder icon then choosing the delete option. You just have to confirm the action from the confirmation popup to delete definitely the selected folder.
```
